src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
```

Comment:
Don't begin class or method implementation with an empty line. Please remove all redundant empty lines at the start of the class definition.